src/main/kotlin/com/pubnub/api/crypto/data/SizedInputStream.kt

```diff
@@ -0,0 +1,44 @@
+package com.pubnub.api.crypto.data
+
+import java.io.InputStream
+
+class SizedInputStream(
+    private val inputStream: InputStream,
+    val size: Int
```

Comment:
I don't see "size" being used anywhere in this class. 
Is it used?

Reply:
It is not. Size is not needed here but it's going to be used elsewhere

Reply:
You know what. It's not important. Either way we will have to materialize this stream eventually if sending it out with uploadfile, but we can do it there, so to make the code simpler I will move computing the size to there